slam-ws/src/nav_goal/bin/movebase_client_map

```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env python
+
+import nav_goal.movebase_client
+import rospy
+import sys
+
+if __name__ == '__main__':
+    rospy.init_node('movebase_client_py')
+
+    if len(sys.argv) < 4:
+        rospy.logerr("Error: Insufficient Arguments - Usage is 'rosrun nav_goal movebase_client x y yaw' where x is distance to move in x, y is distance to move in y, and yaw is rotation in radians.")
+        rospy.signal_shutdown("Error: Insufficient Arguments - Usage is 'rosrun nav_goal movebase_client x y yaw' where x is distance to move in x, y is distance to move in y, and yaw is rotation in radians.")
+    else:
+        try:
+            x = float(sys.argv[1])
+            y = float(sys.argv[2])
+            yaw = float(sys.argv[3])
+        except:
+            rospy.logerr("Error: Incorrect Argument Types - All 3 arguments must be floats or integers.")
+            rospy.signal_shutdown("Error: Incorrect Argument Types - All 3 arguments must be floats or integers.")
+            sys.exit(1)
+
+       	result = nav_goal.movebase_client.movebase_client(x, y, yaw, "map")
+       	rospy.loginfo(result)
```

Comment:
The indentation seems wrong here. Also it is different `bin/movebase_client`:
```
try:
    result = nav_goal.movebase_client.movebase_client(x, y, yaw, "base_link")
    rospy.loginfo(result)
except:
    rospy.logerr("Error - Navigation failed for unknown reasons. Exiting...")
    rospy.signal_shutdown("Error - Navigation failed for unknown reasons. Exiting...")
```

Reply:
The indentation is 4 spaces everywhere I think